### ISSUE TITLE
breakText is broken with character '['

### ISSUE THREAD

**Issue by elinake** [clientIO/joint#1460]
```
var wrappedText = joint.util.breakText('asdfWETUIOPj[JF', {
                width: 100,
});
```

The string above has 'undefined' in the end because I used character '[' and it has more than one line.

http://jsfiddle.net/elinakeranen/jsf479qp/75/

JointJS version: 3.3.1
<img width="145" alt="breakText bug" src="https://user-images.githubusercontent.com/6532597/116053072-f0275180-a682-11eb-8cd4-e47b50bf50d1.png">

**Comment by kumilingus** [comment]
It's fixed in #1461. A workaround:

```js
var wrappedText = joint.util.breakText('asdfWETUIOPj[JF', {
  // size
  width: 100
}, {
  // styles
}, {
  // options
  hyphen: '-'
});
```

**Comment by elinake** [comment]
Thank you so much, that was quick!

**Comment by kumilingus** [comment]
Fixed in `v3.4.0`.
